error: invalid conversion from 'int' to 'CURLoption'

env: windows , qt5.15.2

```

16:48:55: 为项目QDAP执行步骤 ...
16:48:55: 正在启动 "C:\Qt5\Tools\CMake_64\bin\cmake.exe" --build C:/Users/mjc/proj/QDAP/QDAP_libcurl/build/Desktop_Qt_5_15_2_MinGW_64_bit-Debug --target all

[0/2 0.0/sec] Re-checking globbed directories...
[1/30 5.4/sec] Linking CXX executable vendor\yaml-cpp\util\sandbox.exe
[2/30 6.4/sec] Linking CXX executable vendor\yaml-cpp\util\read.exe
[3/30 6.3/sec] Linking CXX executable vendor\yaml-cpp\util\parse.exe
[4/30 5.5/sec] Building CXX object vendor/curlcpp/src/CMakeFiles/curlcpp.dir/curl_easy.cpp.obj
FAILED: vendor/curlcpp/src/CMakeFiles/curlcpp.dir/curl_easy.cpp.obj 
C:\Qt5\Tools\mingw810_64\bin\g++.exe  -IC:/Users/mjc/proj/QDAP/QDAP_libcurl/build/Desktop_Qt_5_15_2_MinGW_64_bit-Debug/vendor/curlcpp/src -IC:/Users/mjc/proj/QDAP/QDAP_libcurl/vendor/curlcpp/src -IC:/Users/mjc/proj/QDAP/QDAP_libcurl/build/Desktop_Qt_5_15_2_MinGW_64_bit-Debug/vendor/curlcpp/src/curlcpp_autogen/include -IC:/Users/mjc/proj/QDAP/QDAP_libcurl/src -IC:/Users/mjc/proj/QDAP/QDAP_libcurl/src/DAP -IC:/Users/mjc/proj/QDAP/QDAP_libcurl/src/config -IC:/Users/mjc/proj/QDAP/QDAP_libcurl/src/ui_components -IC:/Users/mjc/proj/QDAP/QDAP_libcurl/src/device_selector -IC:/Users/mjc/proj/QDAP/QDAP_libcurl/src/devices -IC:/Users/mjc/proj/QDAP/QDAP_libcurl/src/views/input_box_dialog -IC:/Users/mjc/proj/QDAP/QDAP_libcurl/src/views/chips_config_dialog -IC:/Users/mjc/proj/QDAP/QDAP_libcurl/vendor/curlcpp/src/../include -isystem C:/Users/mjc/proj/QDAP/QDAP_libcurl/vendor/curl/include -DQT_QML_DEBUG -g -std=gnu++11 -fdiagnostics-color=always -Wall -Wextra -Wpedantic -MD -MT vendor/curlcpp/src/CMakeFiles/curlcpp.dir/curl_easy.cpp.obj -MF vendor\curlcpp\src\CMakeFiles\curlcpp.dir\curl_easy.cpp.obj.d -o vendor/curlcpp/src/CMakeFiles/curlcpp.dir/curl_easy.cpp.obj -c C:/Users/mjc/proj/QDAP/QDAP_libcurl/vendor/curlcpp/src/curl_easy.cpp
In file included from C:/Users/mjc/proj/QDAP/QDAP_libcurl/vendor/curlcpp/src/curl_easy.cpp:6:
C:/Users/mjc/proj/QDAP/QDAP_libcurl/vendor/curlcpp/include/curl_easy.h:42:36: error: invalid conversion from 'int' to 'CURLoption' [-fpermissive]
     template <> struct option_t<opt> {\
                                    ^
C:/Users/mjc/proj/QDAP/QDAP_libcurl/vendor/curlcpp/include/curl_easy.h:361:9: note: in expansion of macro 'CURLCPP_DEFINE_OPTION'
         CURLCPP_DEFINE_OPTION(CURLOPT_OBSOLETE72, long); /* OBSOLETE, do not use! */
         ^~~~~~~~~~~~~~~~~~~~~
[5/30 6.8/sec] Building CXX object vendor/curlcpp/src/CMakeFiles/curlcpp.dir/curl_cookie.cpp.obj
FAILED: vendor/curlcpp/src/CMakeFiles/curlcpp.dir/curl_cookie.cpp.obj 
C:\Qt5\Tools\mingw810_64\bin\g++.exe  -IC:/Users/mjc/proj/QDAP/QDAP_libcurl/build/Desktop_Qt_5_15_2_MinGW_64_bit-Debug/vendor/curlcpp/src -IC:/Users/mjc/proj/QDAP/QDAP_libcurl/vendor/curlcpp/src -IC:/Users/mjc/proj/QDAP/QDAP_libcurl/build/Desktop_Qt_5_15_2_MinGW_64_bit-Debug/vendor/curlcpp/src/curlcpp_autogen/include -IC:/Users/mjc/proj/QDAP/QDAP_libcurl/src -IC:/Users/mjc/proj/QDAP/QDAP_libcurl/src/DAP -IC:/Users/mjc/proj/QDAP/QDAP_libcurl/src/config -IC:/Users/mjc/proj/QDAP/QDAP_libcurl/src/ui_components -IC:/Users/mjc/proj/QDAP/QDAP_libcurl/src/device_selector -IC:/Users/mjc/proj/QDAP/QDAP_libcurl/src/devices -IC:/Users/mjc/proj/QDAP/QDAP_libcurl/src/views/input_box_dialog -IC:/Users/mjc/proj/QDAP/QDAP_libcurl/src/views/chips_config_dialog -IC:/Users/mjc/proj/QDAP/QDAP_libcurl/vendor/curlcpp/src/../include -isystem C:/Users/mjc/proj/QDAP/QDAP_libcurl/vendor/curl/include -DQT_QML_DEBUG -g -std=gnu++11 -fdiagnostics-color=always -Wall -Wextra -Wpedantic -MD -MT vendor/curlcpp/src/CMakeFiles/curlcpp.dir/curl_cookie.cpp.obj -MF vendor\curlcpp\src\CMakeFiles\curlcpp.dir\curl_cookie.cpp.obj.d -o vendor/curlcpp/src/CMakeFiles/curlcpp.dir/curl_cookie.cpp.obj -c C:/Users/mjc/proj/QDAP/QDAP_libcurl/vendor/curlcpp/src/curl_cookie.cpp
In file included from C:/Users/mjc/proj/QDAP/QDAP_libcurl/vendor/curlcpp/include/curl_cookie.h:33,
                 from C:/Users/mjc/proj/QDAP/QDAP_libcurl/vendor/curlcpp/src/curl_cookie.cpp:8:
C:/Users/mjc/proj/QDAP/QDAP_libcurl/vendor/curlcpp/include/curl_easy.h:42:36: error: invalid conversion from 'int' to 'CURLoption' [-fpermissive]
     template <> struct option_t<opt> {\
                                    ^
C:/Users/mjc/proj/QDAP/QDAP_libcurl/vendor/curlcpp/include/curl_easy.h:361:9: note: in expansion of macro 'CURLCPP_DEFINE_OPTION'
         CURLCPP_DEFINE_OPTION(CURLOPT_OBSOLETE72, long); /* OBSOLETE, do not use! */
         ^~~~~~~~~~~~~~~~~~~~~
[6/30 7.9/sec] Building CXX object vendor/curlcpp/src/CMakeFiles/curlcpp.dir/curl_multi.cpp.obj
FAILED: vendor/curlcpp/src/CMakeFiles/curlcpp.dir/curl_multi.cpp.obj 
C:\Qt5\Tools\mingw810_64\bin\g++.exe  -IC:/Users/mjc/proj/QDAP/QDAP_libcurl/build/Desktop_Qt_5_15_2_MinGW_64_bit-Debug/vendor/curlcpp/src -IC:/Users/mjc/proj/QDAP/QDAP_libcurl/vendor/curlcpp/src -IC:/Users/mjc/proj/QDAP/QDAP_libcurl/build/Desktop_Qt_5_15_2_MinGW_64_bit-Debug/vendor/curlcpp/src/curlcpp_autogen/include -IC:/Users/mjc/proj/QDAP/QDAP_libcurl/src -IC:/Users/mjc/proj/QDAP/QDAP_libcurl/src/DAP -IC:/Users/mjc/proj/QDAP/QDAP_libcurl/src/config -IC:/Users/mjc/proj/QDAP/QDAP_libcurl/src/ui_components -IC:/Users/mjc/proj/QDAP/QDAP_libcurl/src/device_selector -IC:/Users/mjc/proj/QDAP/QDAP_libcurl/src/devices -IC:/Users/mjc/proj/QDAP/QDAP_libcurl/src/views/input_box_dialog -IC:/Users/mjc/proj/QDAP/QDAP_libcurl/src/views/chips_config_dialog -IC:/Users/mjc/proj/QDAP/QDAP_libcurl/vendor/curlcpp/src/../include -isystem C:/Users/mjc/proj/QDAP/QDAP_libcurl/vendor/curl/include -DQT_QML_DEBUG -g -std=gnu++11 -fdiagnostics-color=always -Wall -Wextra -Wpedantic -MD -MT vendor/curlcpp/src/CMakeFiles/curlcpp.dir/curl_multi.cpp.obj -MF vendor\curlcpp\src\CMakeFiles\curlcpp.dir\curl_multi.cpp.obj.d -o vendor/curlcpp/src/CMakeFiles/curlcpp.dir/curl_multi.cpp.obj -c C:/Users/mjc/proj/QDAP/QDAP_libcurl/vendor/curlcpp/src/curl_multi.cpp
In file included from C:/Users/mjc/proj/QDAP/QDAP_libcurl/vendor/curlcpp/include/curl_multi.h:33,
                 from C:/Users/mjc/proj/QDAP/QDAP_libcurl/vendor/curlcpp/src/curl_multi.cpp:6:
C:/Users/mjc/proj/QDAP/QDAP_libcurl/vendor/curlcpp/include/curl_easy.h:42:36: error: invalid conversion from 'int' to 'CURLoption' [-fpermissive]
     template <> struct option_t<opt> {\
                                    ^
C:/Users/mjc/proj/QDAP/QDAP_libcurl/vendor/curlcpp/include/curl_easy.h:361:9: note: in expansion of macro 'CURLCPP_DEFINE_OPTION'
         CURLCPP_DEFINE_OPTION(CURLOPT_OBSOLETE72, long); /* OBSOLETE, do not use! */
         ^~~~~~~~~~~~~~~~~~~~~
ninja: build stopped: subcommand failed.
16:48:56: 进程"C:\Qt5\Tools\CMake_64\bin\cmake.exe"退出，退出代码 1 。
Error while building/deploying project QDAP (kit: Desktop Qt 5.15.2 MinGW 64-bit)
When executing step "构建"
16:48:56: Elapsed time: 00:01.

```

--- a/include/curl_easy.h
+++ b/include/curl_easy.h
@@ -356,13 +356,6 @@ namespace curl  {
         /* Max amount of cached alive connections */
         CURLCPP_DEFINE_OPTION(CURLOPT_MAXCONNECTS, long);
 
-        /* Renamed / obsoleted since 7.37 */
-#if defined(LIBCURL_VERSION_NUM) && LIBCURL_VERSION_NUM > 0x072500
-        CURLCPP_DEFINE_OPTION(CURLOPT_OBSOLETE72, long); /* OBSOLETE, do not use! */
-#else
-        CURLCPP_DEFINE_OPTION(CURLOPT_CLOSEPOLICY, long);
-#endif
-
         /* 73 = OBSOLETE */
 
         /* Set to explicitly use a new connection for the upcoming transfer.

